src/main/java/cz/student/ondrejmal/TaskManager.java

```diff
@@ -1,9 +1,138 @@
 package cz.student.ondrejmal;
 
+import cz.common.ConsoleColors;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class TaskManager {
 
-    public static void main(final String[] args) {
+    public static void main(String[] args) {
+        System.out.println(ConsoleColors.BLUE + "Task Manager Coderslab" + ConsoleColors.RESET);
+        System.out.println("System will now load assigned task files.");
+
+        String filePath = "tasks.csv";
+        ArrayList<String[]> tasks = loadTasks(filePath);
+
+
+        Scanner scanner = new Scanner(System.in);
+
+        printOptions();
+
+        while (true) {
+            String command = scanner.nextLine().trim().toLowerCase();
+            System.out.println("Your command is: " + command.toUpperCase());
+
+            switch (command) {
+                case "add" -> addingTask(tasks, scanner);
+                case "remove" -> removingTask(tasks, scanner);
+                case "list" -> listingTasks(tasks);
+                case "exit" -> programExit(tasks, filePath, scanner);
+                default -> System.out.println("Invalid task selection. Please enter a valid task");
+            }
+        }
+    }
 
+    public static void printOptions() {
+        System.out.println("Please type name of possible operations to proceed.");
+        System.out.println("Select the task from the list below:");
+        System.out.println("1) Add");
+        System.out.println("2) Remove");
+        System.out.println("3) List");
+        System.out.println("4) Exit");
+        System.out.println("Command in not case sensitive.");
     }
 
-}
+    public static ArrayList<String[]> loadTasks(String filepath) {
+        ArrayList<String[]> fileTasks = new ArrayList<>();
+        try (BufferedReader reader = new BufferedReader(new FileReader(filepath))) {
+            String line;
+            while ((line = reader.readLine()) != null)
+                fileTasks.add(line.split(","));
+        } catch (
+                IOException e) {
+            System.err.println("Problem with loading the file. File not found. Check if the file exists.");
```

Comment:
BUG: spatny popis chyby v vyjimce, IOException != vzdy FileNotFound vyjimka. FileNotFoundException je podtrida tridy IOException.

Zdroj: FileNotFoundException

```suggestion
            System.err.println("Failed to load tasks.csv");
```

Reply:
opraveno